scripts/db_migrator.py

```diff
@@ -974,13 +992,24 @@ def version_4_0_2(self):
         if self.stateDB.keys(self.stateDB.STATE_DB, "FAST_REBOOT|system"):
             self.migrate_config_db_flex_counter_delay_status()
 
+        sflow_tbl = self.configDB.get_table('SFLOW')
```

Comment:
Fixed. Pls review. Since submodule of sonic-utilities was not updated in master and 202305, it was not clear that this has to be added as part of new DB version or integrate to existing version.